tritonbench/utils/triton_op.py

```diff
@@ -1337,9 +1337,10 @@ def get_peak_mem(
             torch.cuda.reset_peak_memory_stats()
             torch.cuda.empty_cache()
         if use_cuda_graphs:
-            self.do_bench_cudagraph_mem(
-                fn, n_repeat=2, grad_to_none=grad_to_none, device_type=device_type
-            )
+            with torch.cuda.stream(torch.cuda.Stream()):
```

Comment:
Can we put this context inside `self.do_bench_cudagrpah_mem()` ?  Similar to https://github.com/pytorch-labs/tritonbench/blob/main/tritonbench/components/kineto/trace.py#L29